### ISSUE TITLE
feature request: native android version

### ISSUE THREAD

**Issue by arcade-legend** [LMMS/lmms#6110]
hi

i would like to resquest a native android version of lmms.
this is possible in theory I think when using QT 6, which supports android.

**Comment by Spekular** [comment]
I'm tempted to close this as it's massively out of reasonable scope for the near future and low priority. LMMS's UI isn't suitable for android, and before that can be fixed the UI/core decouple + single window designs should be complete. In other words that's one huge task blocked by two more huge tasks, and then there's porting and performance to deal with too.

Related: 5195

**Comment by rdrpenguin04** [comment]
Also worth noting that Qt 5 supports Android as well, so the point about Qt 6 isn't relevant.

**Comment by rdrpenguin04** [comment]
I will say that an Android version would be fun and useful, but as Spekular said, the LMMS GUI is not suited for mobile whatsoever. After the decoupling, I could see an alternate version being made, and I look forward to that project... in 10 years.

**Comment by arcade-legend** [comment]
Maybe the things required for this can be done very slowly with other GUI's in mind (like Android)?

**Comment by qnebra** [comment]
Very, very slowly. Mobile UI, for any platform, is completly different than desktop UI.

**Comment by arcade-legend** [comment]
PUT IT UP YOUR ASS
